Formula/b/breseq.rb

```diff
@@ -0,0 +1,42 @@
+class Breseq < Formula
+  desc "Computational pipeline for finding mutations in short-read DNA resequencing data"
+  homepage "https://barricklab.org/breseq"
```

Comment:
since it would be redirected to `https://barricklab.org/twiki/bin/view/Lab/ToolsBacterialGenomeResequencing`, might be good to use `https://barricklab.org/twiki/bin/view/Lab/ToolsBacterialGenomeResequencing` instead.